pkg/cli/cvd.go

```diff
@@ -178,3 +231,28 @@ func (o *CVDOutput) String() string {
 		fmt.Sprintf("%s/hosts/%s/cvds/%s/logs/", o.BaseURL, o.Host, o.CVD.Name)
 	return res
 }
+
+const cvdHostPackageName = "cvd-host_package.tar.gz"
+
+func listRequiredImageFilenames(listFilename, androidProductOutDir, androidHostOutDir string) ([]string, error) {
+	var result []string
+	f, err := os.Open(listFilename)
+	if err != nil {
+		return nil, err
+	}
+	defer f.Close()
+	r := bufio.NewReader(f)
+	for {
+		line, err := r.ReadString('\n')
+		line = strings.TrimRight(line, "\n")
```

Comment:
touching `line` without checking `err` doesn't seem correct.

Reply:
good catch! this approach in favor of suggested os.ReadFile